nservicebus/azure-storage-queues/configuration_config_asq_[,6].partial.md

```diff
@@ -0,0 +1,10 @@
+-NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configuration object to specify the endpoint name and select a scale-out option.
+
+Settings can be overridden by adding to the `web.config` or the `app.config` files a configuration section called `AzureQueueConfig`:
```

Comment:
`AzureQueueConfig` is the only way to control the transport in v6 @SimonCropp I'm not sure why removing that would make sense for a document that targets v6 specifically.